vpr/src/draw/draw.cpp

```diff
@@ -998,16 +998,16 @@ static void drawplace(ezgl::renderer* g) {
                 ezgl::color block_color;
                 t_logical_block_type_ptr logical_block_type = nullptr;
 #    ifdef VTR_ENABLE_DEBUG_LOGGING
```

Comment:
Do we need this guard?  Does it only control the breakpoint functionality or something else?  Does compiling in breakpoints slow the tool down?
If not, we can just eliminate this #ifdef throughout the code.

Reply:
I'd factor this into a subroutine; shorter routines with meaningful names are easier to understand and change.